qutip/tests/test_qubitcircuit.py

```diff
@@ -32,9 +32,9 @@
 ###############################################################################
 
 import numpy as np
-from numpy.testing import assert_, assert_allclose, run_module_suite
+from numpy.testing import assert_raises, assert_, assert_allclose, run_module_suite
 from qutip.qip.operations.gates import (
-    gate_sequence_product, rx, expand_operator)
+    gate_sequence_product, rx)
```

Comment:
why is `expand_operator` removed? Cc @BoxiLi.

Reply:
Removed unused import